---a/net/minecraft/world/gen/feature/WorldGenTallGrass.java
+++b/net/minecraft/world/gen/feature/WorldGenTallGrass.java
@@ -1,6 +1,7 @@
 package net.minecraft.world.gen.feature;
 
 import java.util.Random;
+
 import net.minecraft.block.Block;
 import net.minecraft.block.BlockTallGrass;
 import net.minecraft.block.material.Material;
@@ -9,36 +9,31 @@
 import net.minecraft.util.BlockPos;
 import net.minecraft.world.World;
 
-public class WorldGenTallGrass extends WorldGenerator
-{
+public class WorldGenTallGrass extends WorldGenerator {
+
     private final IBlockState field_175907_a;
     private static final String __OBFID = "CL_00000437";
 
-    public WorldGenTallGrass(BlockTallGrass.EnumType p_i45629_1_)
-    {
+    public WorldGenTallGrass(BlockTallGrass.EnumType p_i45629_1_) {
         this.field_175907_a = Blocks.tallgrass.getDefaultState().withProperty(BlockTallGrass.TYPE, p_i45629_1_);
     }
 
-    public boolean generate(World worldIn, Random p_180709_2_, BlockPos p_180709_3_)
-    {
+    public boolean generate(World worldIn, Random p_180709_2_, BlockPos p_180709_3_) {
         Block block;
 
-        while (((block = worldIn.getBlockState(p_180709_3_).getBlock()).getMaterial() == Material.air || block.getMaterial() == Material.leaves) && p_180709_3_.getY() > 0)
-        {
+        while (((block = worldIn.getBlockState(p_180709_3_).getBlock()).getMaterial() == Material.air || block.getMaterial() == Material.leaves) && p_180709_3_.getY() > 0) {
             p_180709_3_ = p_180709_3_.down();
         }
 
-        for (int i = 0; i < 128; ++i)
-        {
+        for (int i = 0; i < 128; ++i) {
             BlockPos blockpos1 = p_180709_3_.add(p_180709_2_.nextInt(8) - p_180709_2_.nextInt(8), p_180709_2_.nextInt(4) - p_180709_2_.nextInt(4), p_180709_2_.nextInt(8) - p_180709_2_.nextInt(8));
 
-            if (worldIn.isAirBlock(blockpos1) && Blocks.tallgrass.canBlockStay(worldIn, blockpos1, this.field_175907_a))
-            {
+            if (worldIn.isAirBlock(blockpos1) && Blocks.tallgrass.canBlockStay(worldIn, blockpos1, this.field_175907_a)) {
                 worldIn.setBlockState(blockpos1, this.field_175907_a, 2);
             }
         }
 
         return true;
     }
-}
+}
